sbt_common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayConcept.scala

```diff
@@ -17,7 +17,7 @@ case class DisplayConcept(
 }
 
 case object DisplayConcept {
-  def apply(concept: Concept): DisplayConcept = DisplayConcept(
+  def apply(concept: AbstractConcept): DisplayConcept = DisplayConcept(
```

Comment:
This is just enough to get the internal models working – I’ll add proper support for QualifiedConcepts in the display models in a separate patch.

Reply:
See #1717.